pkg/plugin/installer/vcs_installer.go

```diff
@@ -149,17 +150,17 @@ func (i *VCSInstaller) solveVersion(repo vcs.Repo) (string, error) {
 
 // setVersion attempts to checkout the version
 func (i *VCSInstaller) setVersion(repo vcs.Repo, ref string) error {
-	debug("setting version to %q", i.Version)
+	slog.Debug("setting version", "version", i.Version)
 	return repo.UpdateVersion(ref)
 }
 
 // sync will clone or update a remote repo.
 func (i *VCSInstaller) sync(repo vcs.Repo) error {
 	if _, err := os.Stat(repo.LocalPath()); os.IsNotExist(err) {
-		debug("cloning %s to %s", repo.Remote(), repo.LocalPath())
+		slog.Debug("cloning", "source", repo.Remote(), "destination", repo.LocalPath())
```

Comment:
On line 163 there is the same key/value pair but the key is "remote". Can we have the same key name everywhere? Please also keep it consistent with line 92.

The idea is that people can use log tools to filter on keys and values. We want to be consistent with the key names to aide these tools.

Reply:
I corrected this across this file for you.